cmd/podman/images/push.go

```diff
@@ -109,7 +109,7 @@ func pushFlags(cmd *cobra.Command) {
 	flags.BoolVar(&pushOptions.TLSVerifyCLI, "tls-verify", true, "Require HTTPS and verify certificates when contacting registries")
 
 	compressionFormat := "compression-format"
-	flags.StringVar(&pushOptions.CompressionFormat, compressionFormat, "", "compression format to use")
+	flags.StringVar(&pushOptions.CompressionFormat, compressionFormat, containerConfig.Engine.CompressionFormat, "compression format to use")
```

Comment:
You should not set this hear, it needs to be set on the server side not the client side.

Reply:
Should be handled here:
https://github.com/containers/podman/blob/main/pkg/domain/infra/abi/images.go#L309-L315

Reply:
I thought we wanted from the client side (so that it reflects in the `podman push --help` output).

I've pushed a new version where it is set in `pkg/domain/infra/abi/images.go`